samples/presentation/retrieve_the_number_of_slides/cs/Program.cs

```diff
@@ -1,37 +1,49 @@
-#nullable disable
-
 using DocumentFormat.OpenXml.Packaging;
 using System;
 using System.Linq;
 
+if (args.Length >= 2)
+{
+    RetrieveNumberOfSlides(args[0], args[1]);
+}
+else
+{
+    RetrieveNumberOfSlides(args[0]);
+}
```

Comment:
```suggestion
if (args is [ {} fileName, {} includeHidden])
{
    RetrieveNumberOfSlides(fileName, includeHidden);
}
else if (args is [ {} fileName ])
{
    RetrieveNumberOfSlides(fileName);
}
```

not needed in this PR, but using pattern matching will clean the samples up a bit and do better error checking

Reply:
also, in a later pr, strings should be pre-parsed for the methods so the methods (such as RetrieveNumberOfSlides) takes the types it actually needs

Reply:
Made this change since I was making other changes.